File: HorseRacing.Contracts/Models/User/Requests/UpdateUser/UpdateUserRequestDto.cs
using HorseRacing.Contracts.Base.Dto;

namespace HorseRacing.Contracts.Models.User.Requests.UpdateUser
{
    public class UpdateUserRequestDto : BaseDto
    {
        /// <summary>
        /// Код пользователя
        /// </summary>
        public Guid UserId { get; set; }
        /// <summary>
		/// Имя
		/// </summary>
		public string FirstName { get; set; } = string.Empty;
        /// <summary>
        /// Фамилия
        /// </summary>
        public string LastName { get; set; } = string.Empty;
        /// <summary>
        /// Фамилия
        /// </summary>
        public string UserName { get; set; } = string.Empty;
        /// <summary>
		/// Электронная почта
		/// </summary>
		public string Email { get; set; } = string.Empty;
        /// <summary>
        /// Телефон
        /// </summary>
        public string Phone { get; set; } = string.Empty;
        /// <summary>
        /// Пароль пользователя
        /// </summary>
        public string Password { get; set; }
    }
}

File: HorseRacing.Web/src/interfaces/api/contracts/model/user/requests/update-user/update-user-request-dto.ts
import type { BaseDto } from '~/interfaces/api/contracts/base/dto/base-dto';

export interface UpdateUserRequestDto extends BaseDto {
    /** Код */
    UserId: string;
    /** Имя */
    FirstName: string;
    /** Фамилия */
    LastName: string;
    /** Логин */
    UserName: string;
    /** Телефон */
    Phone: string;
    /** Почта */
    Email: string;
    /** Пароль */
    Password: string;
}
